packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Transform/Transform.tsx

```diff
@@ -28,30 +27,31 @@ const transformOptions = makeItems([
   'transform-flip-v'
 ])
 
-interface TransformProps
-  extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {}
+interface TransformProps extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {
+  height?: number
+}
 interface TransformCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = TransformProps & TransformCallProps
 
 const Transform = (props: Props) => {
-  const collapseTransform = useMediaQuery({ query: '(max-height: 800px)' })
+  const { height, ...rest } = props
 
-  if (collapseTransform) {
+  if (height && height <= 720) {
```

Comment:
Please move to constants

Reply:
fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)